src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/simple_response_commands_test.cc

```diff
@@ -131,6 +127,36 @@ TYPED_TEST(MobileResponseCommandsTest, Run_SendResponseToMobile_SUCCESS) {
   command->Run();
 }
 
+template <class CammandWithUnsubscribe>
+class MobileResponseWithUnsubscribeCommandsTest
+    : public CommandsTest<CommandsTestMocks::kIsNice> {
+ public:
+  typedef CammandWithUnsubscribe CommandWithUnsubscribeType;
```

Comment:
@IGapchuk why do you need this typedef if your new type name longer than old one? :)

Reply:
@AKalinich-Luxoft Fixed in c65a8ac14947dd4afe2bb1c68bffa2744a5065d2.